crates/block-producer/src/mempool/batch_graph.rs

```diff
@@ -91,7 +91,7 @@ impl BatchGraph {
         &mut self,
         id: BatchJobId,
         transactions: Vec<TransactionId>,
-        parents: BTreeSet<TransactionId>,
+        mut parents: BTreeSet<TransactionId>,
```

Comment:
Could we add an explanation of what `parents` is to the doc comment for this function? (e.g., we should mention that parents may contain transactions also in `transactions`).